### ISSUE TITLE
Remove navigation bar items from HERO sections

### ISSUE THREAD

**Issue by JonasEriksson** [tailwindlabs/tailwindui-issues#729]
Pretty simple: TailwindUI Marketing sections provide individual building blocks - apart from the HERO sections, which all but one have navbar mixed in, yet there are also separate navbars. Why not simply strip out the navbar out of the Hero sections, it seems non-logical and creates lotzs of code to get rid of if someone only wants the ... Hero section part.

**Comment by reinink** [comment]
Hey thanks for this feedback @JonasEriksson! We'll certainly keep this in mind for the future.

For what it's worth, I believe the reason that these hero components have the navigation included is because hero sections like this are typically interacting in some way with the header — for example some components have the background image slide up behind the header. While I totally appreciate how it takes effort to remove the header, it's probably still better to include them, just so that folks know how to integrate the hero components with the header.

Basically, there are design tips here that may not be obvious to everyone, if that makes sense.

I appreciate you taking the time either way to share this feedback!
